spec/models/game_spec.rb

```diff
@@ -3,3 +3,14 @@
 RSpec.describe Game, type: :model do
   pending "add some examples to (or delete) #{__FILE__}"
 end
+
+RSpec.describe Game, type: :model do
+  it "populate board" do
+  	Game.populate_board!
```

Comment:
For this test, since it happens in an `after_create`.

We can test it like...

`game = FactoryGirl.create :game`
Then in the expectations, use `game.pieces.where(type: 'rook').white.x`